### PR TITLE
Fork content field

### PR DESCRIPTION
**Pull request by MadeByMike**
Contains a fork of the keystone content field to provide a better editing experience.

Updates dependencies in Keystone.

Re-organises some directories.

### CODE REVIEW

**Comment by mitchellhamilton** [comment]
You need to add this stuff back: https://github.com/WestpacGEL/GEL/pull/231/files/a6f44f75f05b64d4aba86ec48734745d0d4ef702#diff-fc10a6682269c1fefc2abe05d1500b8a

**Comment by MadeByMike** [comment]
image
@mitchellhamilton am I still missing something?

File: website/field-content/block-components/dist/field-content.esm.js [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/mike/Development/GEL/website/field-content/src/views/index.js
```
Comment (dominikwilkowski) [inline website/field-content/block-components/dist/field-content.esm.js]: This dist should probably be deleted :)

File: website/field-content/block-components/dist/field-content.cjs.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+let unregister = require('/Users/mike/Development/GEL/node_modules/@preconstruct/hook/dist/hook.cjs.js').___internalHook(
+	'/Users/mike/Development/GEL'
+);
+
+module.exports = require('/Users/mike/Development/GEL/website/field-content/src/views/index.js');
```
Comment (dominikwilkowski) [inline website/field-content/block-components/dist/field-content.cjs.js]: Delete me

File: babel.config.js [hunk 3]
```diff
@@ -6,4 +8,26 @@ module.exports = {
...
+				[
+					'@babel/preset-env',
+					{
+						targets: {
+							node: 10,
+							browsers: [
+								'last 2 chrome versions',
+								'last 2 firefox versions',
+								'last 2 safari versions',
+								'last 2 edge versions',
```
Comment (dominikwilkowski) [inline babel.config.js]: still wrong versions but let's fix that later

**Pull request merged**